Week1_final.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<p>Comment on the code to say what it is and let the student know that they don't have to worry about replicating the code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-wildbillcolorado/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>